packages/js-dapi-client/lib/BlockHeadersProvider/index.js

```diff
@@ -0,0 +1,3 @@
+const BlockHeadersProvider = require('./BlockHeadersProvider');
```

Comment:
we are trying to avoid index (read 5 things what I regret about Node.js)

Reply:
Fixed